### ISSUE TITLE
Coding Competitions - If registration not started

### ISSUE THREAD

**Issue by Nayaker** [Nayaker/Algorithmist#35]
- Suppose we have a competition that has been added to the backend but is still to open its registration. We want to show the position of registration status as not started and the time left for it to start. 

- I have recently added two more fields in the <a href="https://script.googleusercontent.com/macros/echo?user_content_key=CaLwBxUfR6tLBmFGO58yv01PbW8nX7NV5U28zmg7Uqy3aG3N8ycTxq6-e-j4d0sulclUjH84Tu3vKK3nK3wU1A0w2UITkBnLm5_BxDlH2jW0nuo2oDemN9CCS2h10ox_1xSncGQajx_ryfhECjZEnKHEc-aLaltP9O2ASeUMZJtNW8cSVOyBKfY-TYR-eBoFpxgG6pQf-WNCoQiDIy2FPoojkiKnZSLlVfrVGTwpv_T6zC8ya0Mestz9Jw9Md8uu&lib=M_adtjhtYqTY4x3CHvLkZEzxNTvjCbw04" target="_blank">API</a>, the registration starting date and the registration starting time of that day. 
  <img width="946" alt="Screenshot 2022-12-01 at 6 17 54 PM" src="https://user-images.githubusercontent.com/93304796/205056802-e45f42c7-0b00-4f23-b75b-2da27372de88.png">

- Basically, you need to implement the logic for the below. 
  <img width="993" alt="Screenshot 2022-12-01 at 6 21 05 PM" src="https://user-images.githubusercontent.com/93304796/205057441-860579b8-2aaf-4430-9c6b-fdd46430875d.png">

**Comment by adrianmarinwork** [comment]
Hello, I can take care of this if you want!

**Comment by Nayaker** [comment]
Great! sure, I am assigning this task to you.

**Comment by adrianmarinwork** [comment]
Hey, when trying to `npm install` the client part of the app I get packages and dependencies errors related to React.
Looks like some of the dependencies need older versions than the ones you have the in project.
Do you know something?

![image](https://user-images.githubusercontent.com/118568289/205170909-71641c2c-31f4-4829-8d02-093ba4983196.png)

**Comment by Nayaker** [comment]
It's a react library dependency issue, Stackoverflow should have some valuable solutions for this there. https://stackoverflow.com/questions/71835697/create-react-app-dependency-version-issues-with-react-18 This might help. 

You can try the below code - 
<img src="https://user-images.githubusercontent.com/93304796/205266509-fb60b4bb-8517-4d55-92c7-70ebe617fcf3.png" height="80px"></img>

**Comment by Ayushpanditmoto** [comment]
> Hey, when trying to `npm install` the client part of the app I get packages and dependencies errors related to React. Looks like some of the dependencies need older versions than the ones you have the in project. Do you know something?
> 
> ![image](https://user-images.githubusercontent.com/118568289/205170909-71641c2c-31f4-4829-8d02-093ba4983196.png)

Try npm i --force

**Comment by Nayaker** [comment]
I am closing this issue. The issue has been resolved by - @adrianmarinwork, excellent work 🚀
